Makefile

```diff
@@ -36,6 +36,10 @@ ifeq ($(LOCAL),1)
 TAGS += local
 endif
 
+ifeq ($(MAKECMDGOALS),minimal)
```

Comment:
I thought we decided on not adding the minimal client option?

Reply:
Nice catch! Forgot to revert those lines.